### PR TITLE
Create contributions instead of proposals

### LINKED ISSUE #26: Create contributions instead of proposals
We can now create ERC721 contributions directly, instead of going through a proposal.

### PR DESCRIPTION
**Pull request by raucao**
Also watch for contributions added and log them.

closes #26

### CODE REVIEW

**Comment by bumi** [comment]
related to the potential noce issue: https://github.com/ethers-io/ethers.js/issues/319

**Comment by raucao** [comment]
I think this is ready to merge, so we can start using it with the new contracts.

I opened #28 for fixing the actual nonce issue.

**Comment by bumi** [comment]
but that "fix" does not block the http request?

**Comment by raucao** [comment]
Nope, see the lines where I changed it to not wait for the promise to resolve.

**Comment by bumi** [comment]
was wondering because of the await before... all that JS magic... :boom:

File: integrations/mediawiki.js [hunk 1]
```diff
@@ -168,10 +172,11 @@ module.exports = async function(robot, kredits) {
...
     fetchChanges()
       .then(res => groupChangesByUser(res))
-      .then(res => createProposals(res))
+      .then(res => createContributions(res))
       .then(() => updateTimestampForNextFetch());
   }
 
-  cron.schedule('0 7 * * *', processWikiChangesSinceLastRun);
+  // cron.schedule('0 7 * * *', processWikiChangesSinceLastRun);
+  processWikiChangesSinceLastRun();
```
Comment (bumi) [inline integrations/mediawiki.js]: I think you accidentally the `cron.schedule`

**Comment by raucao** [comment]
There's no `await` for the handler function before returning the 200. That was wrapped in a `.then` function, which I removed. For example:

https://github.com/67P/hubot-kredits/blob/dcbce66796669c96d931a70f0f2e3ba52437af31/integrations/github.js#L173-L174

**Review by bumi — APPROVED** [review]
the scheduler needs to go back, then let's merge it and work on #28

**Comment by raucao** [comment]
Oops, I missed that comment.

**Pull request merged**